src/components/SecondaryPanes/index.js

```diff
@@ -93,7 +94,27 @@ class SecondaryPanes extends Component {
         : L10N.getStr("breakpoints.disable")
     };
 
-    return <input {...inputProps} />;
+    return (
+       <div className="breakpoints-buttons">
+         {this.renderBreakpointsDropdown()}
+         <input {...inputProps} />
+       </div>
+     );
+  }
+
+   
+
+  renderBreakpointsDropdown() {
```

Comment:
lets add a new feature flag here so that it's off by default for now:

https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#creating-a-new-feature-flag